lib/scout_apm/utils/sql_sanitizer.rb

```diff
@@ -42,7 +42,7 @@ def to_s
       def to_s_postgres
         sql.gsub!(PSQL_PLACEHOLDER, '?')
         sql.gsub!(PSQL_VAR_INTERPOLATION, '')
-        sql.gsub!(PSQL_REMOVE_STRINGS, '?')
+        sql.gsub!(PSQL_AFTER_WHERE) {|c| c.gsub(PSQL_REMOVE_STRINGS, '?')}
```

Comment:
I don't like this name. Every other name is the "what" of the regex, and this name is the "where". Can you reuse the original `PSQL_REMOVE_STRINGS` name instead?

Reply:
`gsub` is probably wrong. I don't think that an SQL query can have multiple where clauses?

Reply:
Subqueries?

Reply:
Valid - but the PR code doesn't work w/ subqueries either.

Reply:
> I don't like this name. Every other name is the "what" of the regex, and this name is the "where". Can you reuse the original `PSQL_REMOVE_STRINGS` name instead?

@cschneid By reusing the original `PSQL_REMOVE_STRINGS` name do you mean to rename this regex to something like `PSQL_REMOVE_STRINGS_AFTER_WHERE`? Or unite "after where" regex with  "remove strings" regex, so   `PSQL_REMOVE_STRINGS` will do both: checks "after where" and then check literal strings?

Reply:
The name is fine after reading the code a bit more, but do add tests to make sure this doesn't break with subqueries, and also that this code fixes the original issue.

Reply:
I added the tests for strings and change regex for the subqueries. Regex do check from `where` statement to the `select` of a subquery or to the end on the string
![4711abcc_2019 25 04](https://user-images.githubusercontent.com/42464638/56750405-62ab8100-678c-11e9-9151-f2c128d74af7.png)
Here is a screenshot of the query from the issue 
![e03d2747_2019 25 04](https://user-images.githubusercontent.com/42464638/56749939-668ad380-678b-11e9-86c0-47c704a8bb39.png)